_includes/analytics.html

```diff
@@ -0,0 +1,11 @@
+{%- if jekyll.environment == 'production' and site.analytics.enabled == true -%}
```

Comment:
analytics are off. any reason you cant just delete the file?

Reply:
analytics are something i figure we might wanna use Eventually:tm: and it doesn't really hurt to include

Reply:
While I'm find analytics fine (and actually prefer to have), I sure gonna wish for something that isn't google analytics :p